corehq/apps/userreports/reports/filters/values.py

```diff
@@ -137,11 +137,18 @@ def to_sql_values(self):
         }
 
 
+class IsDistinctFromFilter(BasicFilter):
+
+    def build_expression(self, table):
+        return get_column(table, self.column_name).is_distinct_from(self.parameter)
+
+
 class NumericFilterValue(FilterValue):
     DBSpecificFilter = namedtuple('DBSpecificFilter', ['sql', 'es'])
     operators_to_filters = {
         '=': DBSpecificFilter(EQFilter, filters.term),
         '!=': DBSpecificFilter(NOTEQFilter, filters.not_term),
+        'distinct from': DBSpecificFilter(IsDistinctFromFilter, filters.not_term),
```

Comment:
does this need to be `is distinct from`

Reply:
This is just how you define it in the report, not the actual query

Reply:
got it. thanks emord